runtime_builders/publish_builders.py

```diff
@@ -34,34 +32,26 @@ def main():
                         required=True)
     args = parser.parse_args()
 
-    manifest = {}
+    manifest = builder_util.load_manifest_file()
```

Comment:
If you want to be fancy with style:

manifest = builder_util.load_manifest_file() or {}